src/components/common/YourCompany/NameTranslations/AddNameTranslation.vue

```diff
@@ -6,46 +6,39 @@
     />
     <!-- Name Translation form -->
     <v-form v-model="nameTranslationForm" ref="nameTranslationForm" class="name-translation-form">
-      <v-row>
-        <v-col class="pb-0">
```

Comment:
There is no need to use rows/columns here, since this is a standalone component (no page level formatting).

Before:

![image](https://user-images.githubusercontent.com/10002889/202011328-b3db7973-d76a-42be-a479-bb9b63f6183a.png)

After:

![image](https://user-images.githubusercontent.com/10002889/202011402-a0cbcba8-8c77-4ff3-bc6f-dfce9e3e2d09.png)

Reply:
This diff looks better if you go to ⚙️ dropdown above and check "Hide whitespace" (ie, indentation changes).